SleepEngine/Engine/Utils/json_serialize_utils.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+BEGIN_NAMESPACE_SLEEP
+
+static void from_json(nlohmann::json const& json, glm::vec2& vec)
+{
+    vec.x = json.at("x");
+    vec.y = json.at("y");
+}

```

Comment:
[Optional] I would write `json.at("x").get_to(vec.x);` instead.

Reply:
why?

Reply:
No but on my opinion it is more beautiful